docs/src/man/split_apply_combine.md

```diff
@@ -6,10 +6,24 @@ framework for handling this sort of computation is described in the paper
 "[The Split-Apply-Combine Strategy for Data Analysis](http://www.jstatsoft.org/v40/i01)",
 written by Hadley Wickham.
 
-The DataFrames package supports the split-apply-combine strategy through the `by`
-function, which is a shorthand for `groupby` followed by `map` and/or `combine`.
-`by` takes in three arguments: (1) a `DataFrame`, (2) one or more columns to split
-the `DataFrame` on, and (3) a specification of one or more functions to apply to
+The DataFrames package supports the split-apply-combine strategy through the
+`combine`, `select`/`select!` and `transform`/`transform!` functions.
+
+In order to perform operations by groups you first need to create a `GroupedDataFrame`
```

Comment:
I would put groupeddataframe at the end. The beauty of the current syntax is that one can define transform/select/combine and then say “and by the way, they are all defined on groupedDataFrame”

Reply:
The problem with the current manual is that this is split_apply_combine.md section.